src/mapping/query/functions.rs

```diff
@@ -49,24 +140,54 @@ impl Function for ToStringFn {
                 _ => Value::Bytes(v.as_bytes()),
             }),
             Err(err) => match &self.default {
-                Some(v) => Ok(v.clone()),
+                Some(v) => v.execute(ctx),
                 None => Err(err),
             },
         }
     }
+
+    fn parameters() -> &'static [Parameter] {
+        &[
+            Parameter {
+                keyword: "value",
+                kinds: &[],
```

Comment:
I decided to use an empty slice as "anything goes", since "nothing goes" is never really something we need. We can change this in the future if we have to, by nesting this into an enum for example.

Reply:
Agreeed. Tackled in 12af05d.